docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -212,14 +246,36 @@ properties must be defined.
 The type of kNN algorithm to use. Currently only `hnsw` is supported.
 
 `m`:::
-(Required, integer)
+(Optional, integer)
 The number of neighbors each node will be connected to in the HNSW graph.
 Defaults to `16`.
 
 `ef_construction`:::
-(Required, integer)
+(Optional, integer)
 The number of candidates to track while assembling the list of nearest
 neighbors for each new node. Defaults to `100`.
+
+`quantization_options`:::
+(Optional, object)
+An optional section that configures the quantization configuration. The
+quantization configuration is used to reduce the memory footprint of the
+index. Only `byte` quantization is currently supported and can only be configured if
```

Comment:
I get confused by this; " memory footprint of the index".  Quantization retains the original floats, and also has the quantized byte values, so the index is 1.25 times the size, no?   It's really just a "approx knn searches only result in loading the quantized vectors", right ?

Reply:
Memory footprint for approximate search is lower. The disk footprint is now higher. I can adjust the docs to be clearer.

Reply:
@benwtrent 
In the context of the current implementation of quantization in Elasticsearch, if we were to store only the quantized byte vectors and not retain the original float32 vectors, would it be correct to assume that both disk and memory footprints could be reduced to approximately 25% of the original size? This approach seems like it could offer significant savings. Could you elaborate on why this option is not currently supported? Are there plans to introduce such a feature in future releases? Thank you for your insights.